packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -386,8 +401,9 @@ export async function activate(context: vscode.ExtensionContext) {
   // register org picker commands and set up filewatcher for defaultusername
   const orgList = new OrgList();
   await orgList.displayDefaultUsername();
-  context.subscriptions.push(registerOrgPickerCommands(context, orgList));
-
+  context.subscriptions.push(registerOrgPickerCommands(orgList));
+  await setupOrgBrowser(context);
```

Comment:
I think we should change the way we initialize the org browser and org picker because right now both of them are calling for the same info during activation (defaultUsernameOrAlias). 

We should probably also comment out this since we're executing code during activation that is not available, I want to make sure activation is as streamlined as possible.

Reply:
That makes sense, I was thinking about making a call to retrieve defaultUsernameOrAlias here and just passing the info to org picker + browser. I can make that change as part of this PR if needed.

Reply:
I'm ok if you want to do the refactoring of initializing org picker & browser on a follow up PR. I just want to make sure that this PR does not add code that's not going to be accessible as part of the extension's `activate` phase. 

You could comment `await setupOrgBrowser(context)` or modify it so that nothing inside of `setupOrgBrowser` is executed.